src/components/TodoItem/TodoItem.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const TodoItem = ({
+  userId,
+  completed,
+  title,
+  selectedUserId,
+  selectUser,
+}) => (
+  <>
+    <label>
+      <input type="checkbox" checked={completed} readOnly />
+      <p>{title}</p>
+    </label>
+
+    <button
+      className={classNames('button TodoList__user-button', {
+        // eslint-disable-next-line max-len
+        'TodoList__user-button--selected': userId === selectedUserId,
```

Comment:
отключение еслинта удаляем, фантазию включаем)
```suggestion
        'TodoList__user-button--selected': (
          userId === selectedUserId
        ),
```